### DISCUSSION TITLE
Zehirli şişeler sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#287] [Zeka Sorusu]
Önümüzde bulunan beş şişeden birinde şerbet diğer dördünde zehir vardır. Şişelerin etiketleri üzerinde yazılı ifadelerden yalnızca biri doğrudur ve içeriği şerbet olan şişeye aittir. Etiketler şöyledir:
X: Y'nin etiketinde yazan doğrudur.
Y: U ile aynı sıvıdır.
Z: Şerbet burada veya V'dedir.
U: X'tekinden farklı bir sıvıdır.
V: Y ve U dakilerden farklı sıvıdır.
Şerbet hangi şişededir?

**Comment by yigidi** [comment]
X: Y'nin etiketinde yazan doğrudur.
Bu ifade doğru ise, X'te yazan doğru olmalıdır. Bu durumda X, şerbetin olduğu şişeyi etiketler. Şerbet X'te olabilir.

Y: U ile aynı sıvıdır.
Bu ifade doğruysa, Y ve U aynı sıvıya sahip olmalıdır. Ancak U, X'tekinden farklı bir sıvıdır. Bu durumda Y'nin etiketinde yanlış bir ifade olur. Y'de şerbet olamaz.

Z: Şerbet burada veya V'dedir.
Bu ifadeye göre, Z'de şerbet olabilir veya V'de olabilir. Ancak, diğer ifadelerle birlikte değerlendirdiğimizde Z'de şerbet olduğunu belirleyemeyiz.

U: X'tekinden farklı bir sıvıdır.
Bu ifadeye göre, U farklı bir sıvıdır. Şerbetin X'te olduğunu belirledik, bu yüzden U'da şerbet olamaz.

V: Y ve U dakilerden farklı sıvıdır.
Bu ifadeye göre, V farklı bir sıvıdır. Y'de ve U'da şerbet olamadığını belirledik, bu yüzden V'de şerbet olamaz.

Sonuç olarak, analiz ettiğimiz ifadelerin ardından, şerbetin X'te olduğunu belirleyebiliriz.

**Reply by sonsuzus** [reply]
Ama x de şerbet varsa u da zehir vardır. Ve u nun üstünde yazan doğru olmaz mı o zaman?

**Comment by akgembi12** [comment]
Sadece bir ifade doğru olduğuna göre x yanlıştır Demek ki x de y de zehirdir Eğer y yanlışsa u'daki ile aynı sıvı değildir o zaman şerbet de u'ya kalıyor. Bu kombinasyon da bütün şartları sağlıyor.
Şerbet = U

**Comment by Sevvalgocgun1** [comment]
Şerbet Z şişesindedir. Çünkü sadece Z'nin etiketindeki ifade doğru olabilir.
